config/initializers/string_to_sort_atoms.rb

```diff
@@ -0,0 +1,52 @@
+class SmartSortAtom
```

Comment:
Please add a short description of what this class is supposed to be doing and why we need it.